### ISSUE TITLE
Version 3.4.0 failing to install - Unable to find requirements.txt

### ISSUE THREAD

**Issue by dkav** [rpy2/rpy2#764]
Pip install fails with the following error. Using brew installed Python 3.9 on macOS 10.15.7.

`pip install rpy2  `                                        

```
Collecting rpy2
  Using cached rpy2-3.4.0.tar.gz (185 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/local/opt/python@3.9/bin/python3.9 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/zn/l8w7w5pj55z3nqxjmg8zd3j40000gn/T/pip-install-p18adqbi/rpy2_b8c8e1a0192149acb2136cd733942044/setup.py'"'"'; __file__='"'"'/private/var/folders/zn/l8w7w5pj55z3nqxjmg8zd3j40000gn/T/pip-install-p18adqbi/rpy2_b8c8e1a0192149acb2136cd733942044/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/zn/l8w7w5pj55z3nqxjmg8zd3j40000gn/T/pip-pip-egg-info-3mutawyo
         cwd: /private/var/folders/zn/l8w7w5pj55z3nqxjmg8zd3j40000gn/T/pip-install-p18adqbi/rpy2_b8c8e1a0192149acb2136cd733942044/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/zn/l8w7w5pj55z3nqxjmg8zd3j40000gn/T/pip-install-p18adqbi/rpy2_b8c8e1a0192149acb2136cd733942044/setup.py", line 182, in <module>
        with open(
    FileNotFoundError: [Errno 2] No such file or directory: '/private/var/folders/zn/l8w7w5pj55z3nqxjmg8zd3j40000gn/T/pip-install-p18adqbi/rpy2_b8c8e1a0192149acb2136cd733942044/requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by lgautier** [comment]
Ooops. Fixed with 73c2ff7029129ce6395117f361d6a636114a90a1 and b8fb3c500728bb87bac90660c223e317d7b93ab4.
rpy2-3.4.1 will be uploaded in few minutes.

**Comment by lgautier** [comment]
Uploaded. Thanks for reporting the issue quickly.
